terminal/terminal_test.go

```diff
@@ -0,0 +1,73 @@
+package terminal
+
+import (
+	"testing"
+	"runtime"
+
+	"github.com/derekparker/delve/config"
+)
+
+func TestSubstitutePath(t *testing.T) {
+	type cases []struct {
+		rules [][]string
+		path  string
+		res   string
+	}
+
+	casesUnix := cases{
```

Comment:
Maybe do
... := []case{
instead. Code reader can see that it is just an array.